lib/zold/commands/alias.rb

```diff
@@ -0,0 +1,37 @@
+require 'slop'
+require 'rainbow'
+require_relative 'args'
+require_relative '../log'
+
+module Zold
+  # Command to set an alias for wallet ID
+  class Alias
+    def initialize(wallets:, remotes:, log: Log::Quiet.new)
+      @wallets = wallets
+      @remotes = remotes
+      @log = log
+    end
+
+    def run(args = [])
+      opts = Slop.parse(args, help: true, suppress_errors: true) do |o|
+        o.banner = "Usage: zold alias args
+    #{Rainbow('alias set wallet alias').green}
```

Comment:
@botanicus indentation is fucked up, it's better to use a here document.

Reply:
I agree. It’s in all the commands though, so I just reported it as a bug, since it needs adjusting everywhere https://github.com/zold-io/zold/issues/320